packages/crayons-i18n/i18n/en.js

```diff
@@ -8,6 +8,16 @@ export default {
     add: 'Add',
     cancel: 'Cancel',
   },
+  file_uploader: {
```

Comment:
please use `camelCase` for keys instead of `kebab_case`

Reply:
Updated.